source/Core/Exception/StandardException.php

```diff
@@ -23,13 +23,17 @@
 namespace OxidEsales\Eshop\Core\Exception;
 
 use oxRegistry;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LoggerInterface;
 
 /**
  * Basic exception class
  *
  */
-class StandardException extends \Exception
+class StandardException extends \Exception implements \Psr\Log\LoggerAwareInterface
```

Comment:
Add another `use Psr\Log\LoggerAwareInterface` so you can write `implements LoggerAwareInterface` here